Liang-README HW1.md

```diff
@@ -0,0 +1,9 @@
+## General Introduction about Myself
+My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
+
+### Random list about my favorite cafe on campus
```

Comment:
You don't need to increment the numbers yourself. That will happen when the document is rendered. For instance,

```
1. Ground of Being
1. Starbuck on 55TH
1. Pret
1. Dollop
1. Ex-Lib
```

becomes

1. Ground of Being
1. Starbuck on 55TH
1. Pret
1. Dollop
1. Ex-Lib